app/kudos/views.py

```diff
@@ -723,18 +723,27 @@ def redeem_bulk_coupon(coupon, profile, address, ip_address, save_addr=False):
         return None, error, None
     else:
 
-        if profile.bulk_transfer_redemptions.filter(coupon=coupon).exists():
-            error = f'You have already redeemed this kudos.'
-            return None, error, None
+        #if profile.bulk_transfer_redemptions.filter(coupon=coupon).exists():
```

Comment:
Do we actually want to remove this check or were these comments accidentally left in?

Reply:
Oh that should be there :P